taxi/templatetags/query_transform.py

```diff
@@ -0,0 +1,15 @@
+from django import template
+
+register = template.Library()
+
+
+@register.simple_tag
+def query_transform(request, **kwargs):
+    updated = request.GET.copy()
+    for key, value in kwargs.items():
+        if value is not None:
+            updated[key] = value
+        else:
```

Comment:
We can write it without else

Reply:
I can't rewrite without if-else, it doesn't work. And it's not my code, there was such code in video tutorial